components/textfield/index.css

```diff
@@ -131,207 +460,233 @@ governing permissions and limitations under the License.
-      );
-    }
-    .is-valid& {
-      padding-inline-end: calc(
-        var(--spectrum-textfield-texticon-success-icon-width) +
-          var(--spectrum-textfield-quiet-texticon-success-icon-margin-left)
-      );
+    /* The opacity must be set to 1 */
+    opacity: 1;
+
+    &::placeholder {
+      color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-textfield-text-color-disabled)));
     }
   }
-}
-
-.spectrum-Textfield-validationIcon {
-  position: absolute;
-  pointer-events: all;
 
+  /****** Input - Quiet 🤫 ******/
   .spectrum-Textfield--quiet & {
-    padding-inline-end: 0;
-  }
+    border-block-start-width: 0;
+    border-inline-width: 0;
+    margin-block-end: var(--mod-textfield-spacing-block-quiet, var(--spectrum-textfield-spacing-block-quiet));
+    padding-inline: var(--mod-textfield-spacing-inline-quiet, var(--spectrum-textfield-spacing-inline-quiet));
+    background-color: transparent;
+    border-radius: 0;
 
-  .spectrum-Textfield.is-invalid & {
-    inline-size: var(--spectrum-textfield-texticon-invalid-icon-width);
-    block-size: var(--spectrum-textfield-texticon-invalid-icon-height);
-    inset-block-end: calc(
-      calc(var(--spectrum-textfield-texticon-height) / 2) -
-        calc(var(--spectrum-textfield-texticon-invalid-icon-height) / 2)
-    );
+    /* Treat all quiet inputs and text areas the same */
+    resize: none;
+    overflow-y: hidden;
+   }
 
-    inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, var(--spectrum-textfield-texticon-invalid-icon-margin-left));
+  /****** Input - Quiet 🤫 + Disabled 🚫 ******/
+  .spectrum-Textfield--quiet.is-disabled &,
+  .spectrum-Textfield--quiet.is-disabled:hover &,
+  &:disabled {
+    background-color: transparent;
+    border-color: var(--mod-textfield-border-color-disabled, var(--spectrum-textfield-border-color-disabled));
+    color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-textfield-text-color-disabled)));
 
-    .spectrum-Textfield--quiet& {
-      inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, 0);
+    &::placeholder {
+      color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-textfield-text-color-disabled)))
     }
   }
 
-  .spectrum-Textfield.is-valid & {
-    inline-size: var(--spectrum-textfield-texticon-success-icon-width);
-    block-size: var(--spectrum-textfield-texticon-success-icon-height);
-    inset-block-end: calc(
-      calc(var(--spectrum-textfield-texticon-height) / 2) -
-        calc(var(--spectrum-textfield-texticon-success-icon-height) / 2)
-    );
-    inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, var(--spectrum-textfield-texticon-success-icon-margin-left));
+  /****** Input ReadOnly ******/
+  .spectrum-Textfield.is-readOnly &,
+  .spectrum-Textfield.is-readOnly:hover &,
+  &:read-only {
```

Comment:
Same question here re: the pseudo & class - is it for browser compat?

Reply:
I'd say this is similar to my [other comment](https://github.com/adobe/spectrum-css/pull/1544#discussion_r1132776154) about the pseudo-class selectors.